docs/docs/getting-started/installation.md

```diff
@@ -76,6 +76,19 @@ bastionlab_server.stop(srv)
 docker run -p 50056:50056 -d mithrilsecuritysas/bastionlab:latest
 ```
 
+### Configurating the Docker image
+If you want to modify the server's configuration, you should rather run the container like so:

```

Comment:
Replace by:
_I you want to modify the server's configuration, you should run the container with the following code block:_

Reply:
Why do you need to run it like this _instead_ of another way?